### PR TITLE
getType() with detect sensor type

### PR DESCRIPTION
**Pull request by budulinek**

### CODE REVIEW

**Comment by RobTillaart** [comment]
OK, I will review again ...

File: dhtnew.cpp [hunk 1]
```diff
@@ -63,6 +64,12 @@ DHTNEW::DHTNEW(uint8_t pin)
   #endif
 };
 
+uint8_t DHTNEW::getType()
+{
+  if (_type == 0) read();
```
Comment (RobTillaart) [inline dhtnew.cpp:69]: should call **_read()** ?

Comment (RobTillaart) [reply dhtnew.cpp:69]: NO, this is OK.

**Comment by RobTillaart** [comment]
Looks good!

**Comment by budulinek** [comment]
I think that 
`  if (_type == 0) read();`
is fine. read() knows how to handle situation when _type == 0

**Pull request merged**